packages/postgres-database/src/simcore_postgres_database/models/projects.py

```diff
@@ -69,5 +69,11 @@ class ProjectType(enum.Enum):
         server_default="{}",
     ),
     sa.Column("dev", JSONB, nullable=False, server_default=sa.text("'{}'::jsonb")),
+    sa.Column(
```

Comment:
The idea of having a ``quality`` section for the project is fine but again, adding json columns like this sounds will give us troubles in a near future.
I cannot give you a solution right now (created #2080) but in the meantime, it would be very useful if you can provide us with a schema of the content of quality as mentioned above.

Reply:
As already mentioned in several meetings, for the moment the validation is done only in the frontend, and it follows this schema:
https://github.com/ITISFoundation/osparc-simcore/blob/a8a6c592bc15d76f82f020f00536452ac0302705/services/web/client/source/resource/form/resource-quality.json